### PR TITLE
dotCMS/core#17161 Fallback to "no license" screen for forms

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/portlets/dot-form-builder/dot-form-builder.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
```
Comment (alfredo-dotcms) [inline src/app/portlets/dot-form-builder/dot-form-builder.component.spec.ts]: remove

File: src/app/portlets/dot-form-builder/resolvers/dot-form-resolver.service.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            .getMessages([
+                'Forms-and-Form-Builder',
+                'Forms-and-Form-Builder-in-Enterprise',
+                'Learn-more-about-dotCMS-Enterprise',
+                'Contact-Us-for-more-Information',
+                'Still-use-legacy-form-builder'
+            ])
+            .pipe(
+                map((messages: { [key: string]: string }) => {
+                    return {
```
Comment (alfredo-dotcms) [inline src/app/portlets/dot-form-builder/resolvers/dot-form-resolver.service.ts]: aren't you missing the `image path` also?

Comment (alfredo-dotcms) [reply src/app/portlets/dot-form-builder/resolvers/dot-form-resolver.service.ts]: image

Comment (fmontes) [reply src/app/portlets/dot-form-builder/resolvers/dot-form-resolver.service.ts]: Images out, added an icon.

Comment (fmontes) [reply src/app/portlets/dot-form-builder/resolvers/dot-form-resolver.service.ts]: No image now, added an icon

**Pull request merged**